bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```diff
@@ -177,7 +219,7 @@ spec:
               command:
                 - bash
                 - -ec
-                - PGPASSWORD=${PGPOOL_POSTGRES_PASSWORD} psql -U {{ (include "postgresql-ha.postgresqlUsername" .) | quote }} {{- if not (empty (include "postgresql-ha.postgresqlDatabase" .)) }} -d {{ (include "postgresql-ha.postgresqlDatabase" .) | quote }}{{- end }} -h 127.0.0.1 -tA -c "SELECT 1" >/dev/null
+                - PGPASSWORD=${PGPOOL_POSTGRES_PASSWORD} psql -U {{ (include "postgresql-ha.postgresqlUsername" .) | quote }} {{- if not (empty (include "postgresql-ha.postgresqlDatabase" .)) }} -d {{ (include "postgresql-ha.postgresqlDatabase" .) | quote }}{{- end }} -h /opt/bitnami/pgpool/tmp -tA -c "SELECT 1" >/dev/null
```

Comment:
From http://www.ing.iac.es/~docs/external/postgres/html/app-psql.html

```
-h hostname
--host hostname
Specifies the host name of the machine on which the server is running. If the value begins with a slash, it is used as the directory for the Unix-domain socket.
```
I want the healthcheck to enter using the `local` pg_hba entry (using 127.0.0.1 is considered as `host`, not `local`), otherwise it will generate an annoying error message in the logs (every time it tries to access) because it will try authenticating using the certificate

```
pgpool_1  | 2020-09-09 14:40:34: pid 168: ERROR:  CERT authentication failed
pgpool_1  | 2020-09-09 14:40:34: pid 168: DETAIL:  no valid certificate presented
```

That folder is where the the UNIX domain socket is.